src/fundus/publishers/kr/__init__.py

```diff
@@ -0,0 +1,18 @@
+from fundus.publishers.base_objects import Publisher, PublisherGroup
+from fundus.scraping.url import RSSFeed, Sitemap
+
+from .mbn import MBNParser
+
+
+class KR(metaclass=PublisherGroup):
+    default_language = "kr"
```

Comment:
```suggestion
    default_language = "ko"
```
I think the ISO 639-1 language code for Korean should be ˋkoˋ